namespaces/blueprint-parse-result.md

```diff
@@ -0,0 +1,119 @@
+# Blueprint Parser Result Namespace
+
+This document defines elements needed for holding the result of parsing
+API Blueprint. This document uses [API Namespace][].
```

Comment:
So far everything I see here is generic enough that it could probably be used by other parsers. I think we should remove the API Blueprint specific title/descriptions.